etc/aqd.conf.noms

```diff
@@ -19,6 +23,18 @@ user_list_location =
 
 ###############################################################################
 
+[database]
+#database_section = database_postgresql
+database_section = database_sqlite
+
+[database_postgresql]
+dbname = aquilon
+
+[database_sqlite]
+dbfile = /pdisk/aquilon/db/aquilon.db
```

Comment:
maybe it should be relative path to quattordir as in default conf?

Reply:
@urbonegi sure, you are perfectly right! A too quick copy/paste.